test/tall/regression/0700/0704.unit

```diff
@@ -22,15 +22,17 @@ void main() {
   test("synchronized files are up-to-date", () {
     ({
       'lib/src/visitor/async_evaluate.dart': 'lib/src/visitor/evaluate.dart',
-      'lib/src/async_environment.dart': 'lib/src/environment.dart'
+      'lib/src/async_environment.dart': 'lib/src/environment.dart',
     }).forEach((sourcePath, targetPath) {
       var source = new File(sourcePath).readAsStringSync();
       var target = new File(targetPath).readAsStringSync();
 
       var hash = sha1.convert(utf8.encode(source));
       if (!target.contains("Checksum: $hash")) {
-        fail("$targetPath is out-of-date.\n"
-            "Run pub run grinder to update it.");
+        fail(
+          "$targetPath is out-of-date.\n"
+          "Run pub run grinder to update it.",
```

Comment:
Do we only indent string literal continuations in collections? Or is that not yet implemented for the tall style?

Reply:
It uses the same logic as the old formatter where it doesn't indent them so that the strings are all nicely lined up when doing so is unlikely to be confusing but will indent them otherwise. The logic is [here](https://github.com/dart-lang/dart_style/blob/main/lib/src/ast_extensions.dart#L371).

In this case, since there's only one argument that's a string, it doesn't indent the subsequent lines. (I also think the new style helps here because now the argument list splits before the first line too so that the strings actually line up.)